src/main/webapp/app/pages/curation/collapsible/mutation-collapsible/MutationCollapsibleTitle.tsx

```diff
@@ -0,0 +1,77 @@
+import DefaultBadge from 'app/shared/badge/DefaultBadge';
+import InfoIcon from 'app/shared/icons/InfoIcon';
+import { Alteration, StringMutationInfo } from 'app/shared/model/firebase/firebase.model';
+import { getAlterationName, isFlagEqualToIFlag } from 'app/shared/util/firebase/firebase-utils';
+import { componentInject } from 'app/shared/util/typed-inject';
+import { buildAlterationName, parseAlterationName } from 'app/shared/util/utils';
+import { IRootStore } from 'app/stores';
+import { observer } from 'mobx-react';
+import React from 'react';
+import * as styles from './styles.module.scss';
+
+export interface IMutationCollapsibleTitle extends StoreProps {
+  name: string | undefined;
+  mutationAlterations: Alteration[] | null | undefined;
+  stringMutationInfo: StringMutationInfo | null;
+}
+const MutationCollapsibleTitle = ({ name, mutationAlterations, stringMutationInfo, flags }: IMutationCollapsibleTitle) => {
+  const defaultName = 'No Name';
+  let stringMutationBadges;
+  if (stringMutationInfo?.flags && flags) {
+    let tooltipOverlay: JSX.Element | undefined = undefined;
+    stringMutationBadges = (
+      <div className={styles.flagWrapper}>
+        {stringMutationInfo.flags.map(flag => {
```

Comment:
@zhx828  Should we add the grey wrapper if there is only 1 string mutation name?
Maybe add grey wrapper when  `flags.length > 1 || flags.length === 1 && comment !== ''` ?

Reply:
Also, I think I should disable string mutation name comment if there is no name selected. This should cover all the cases.

Reply:
Agree with both.